typings/internalBinding/os.d.ts

```diff
@@ -0,0 +1,21 @@
+declare function InternalBinding(binding: 'os'): {
+  getHostname(ctx: object): string | undefined;
+  getLoadAvg(array: Float64Array): void;
+  getUptime(): number;
+  getTotalMem(): number;
+  getFreeMem(): number;
+  getCPUs(): Array<string | number>;
```

Comment:
```suggestion
  getCPUs(): (string | number)[];
```
for consistency with other dts

Reply:
I have a preference to use `type[]` for simple types and `Array<types>` for more complex ones